build/sbt

```diff
@@ -62,7 +63,7 @@ Usage: $script_name [options]
   -sbt-rc                   use an RC version of sbt
   -sbt-snapshot             use a snapshot version of sbt
 
-  # java version (default: java from PATH, currently $(java -version 2>&1 | grep version))
+  # JAVA_HOME (this will override default JAVA_HOME)
```

Comment:
I'm not sure I understand your new comment. What is this telling users? it overrides JAVA_HOME, but it says JAVA_HOME overrides JAVA_HOME .. ?

Reply:
Well, my words are not clear. I will rollback to old hint info.